data/Makefile

```diff
@@ -839,6 +858,26 @@ min_path_size=200
 %.overlap.n$n.k3.mol.backbone.map.$(ref).n10.paf.gz: %.overlap.n$n.k3.mol.backbone.path %.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
 	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
 
+# Map the reference to the (1:bc / 2:bc) backbone graph and output PAF.
+%.overlap.n$n.mol.mol2.backbone.map.$(ref).n10.paf.gz: %.overlap.n$n.k3.mol.mol2.backbone.path %.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
+	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
+
+# Map the reference to the (1:bc / 2:bc+cos+sqcos) backbone graph and output PAF.
+%.overlap.n$n.mol.mol2-bcs.backbone.map.$(ref).n10.paf.gz: %.overlap.n$n.k3.mol.mol2.backbone.path %.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
+	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
+
+# Map the reference to the (1:bc / 2:bc+k3+cos+sqcos) backbone graph and output PAF.
+%.overlap.n$n.mol.mol2-bkcs.backbone.map.$(ref).n10.paf.gz: %.overlap.n$n.k3.mol.mol2.backbone.path %.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
+	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
+
+# Map the reference to the (1:k3 / 2:bc+cos+sqcos) backbone graph and output PAF.
+%.overlap.n$n.k3.mol.mol2-bcs.backbone.map.$(ref).n10.paf.gz: %.overlap.n$n.k3.mol.mol2.backbone.path %.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
+	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
+
+# Map the reference to the (1:k3 / 2:bc+k3+cos+sqcos) backbone graph and output PAF.
+%.overlap.n$n.k3.mol.mol2-bkcs.backbone.map.$(ref).n10.paf.gz: %.overlap.n$n.k3.mol.mol2.backbone.path %.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
+	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
+
```

Comment:
Is it possible to reduce the redundancy in rules here? Perhaps by using the wild card up to `backbone` and specifying the name for the `.tsv` file?

Reply:
good point, umm, then I can change it to:
`%.map.$(ref).n10.paf.gz: %.path $(lr).k$k-w$w.n100-5000.c2-x.physlr.tsv $(name)/$(ref).k$k-w$w.physlr.tsv`
so the only problem is that then we will need multiple lines for other steps upstream (if it is `c2-x` or not) which is less redundant and we also need to define `Makefile` variables for n100-5000 too. Is that fine?

Reply:
Hmmm yes so it would only be the `c2-x` that could vary? Can you remind me what are the possibilities for that value? For our runs does that vary a lot?

Reply:
I cannot recall well but I guess the other alternative is not to make `c2-x`, just skipping it... but we don't do that now so no variations in our current pipeline.

Reply:
Done

Reply:
@lcoombe  I don't think we have anything set up to change `c2-x`. It's always `c2-x`

Reply:
Ok cool -- that's what I thought - Just wanted to check!

Reply:
I do however plan to add that into the code since there will be times where we need to change the C value, cough *spruce* cough